oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java

```diff
@@ -266,6 +270,39 @@ private String buildSubjectToken(AwsRequestSignature signature)
     return URLEncoder.encode(token.toString(), "UTF-8");
   }
 
+  private boolean canRetrieveRegionFromEnvironment() {
+    // The AWS region can be provided through AWS_REGION or AWS_DEFAULT_REGION. Only one is
+    // required.
+    List<String> keys = ImmutableList.of("AWS_REGION", "AWS_DEFAULT_REGION");
+    for (String env : keys) {
+      String value = getEnvironmentProvider().getEnv(env);
+      if (value != null && !value.isEmpty()) {
+        // Region available.
+        return true;
+      }
+    }
+    return false;
+  }
+
+  private boolean canRetrieveSecurityCredentialsFromEnvironment() {
+    // Check if both AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are available.
+    List<String> keys = ImmutableList.of("AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY");
```

Comment:
Make these env var names constants since they are used in multiple places now and maybe can refer to these constants in tests as well.

Reply:
I made them constant but did not change the tests. If someone changes them its better that the tests fail.